src/main/resources/bidder-config/limelightDigital.yaml

```diff
@@ -0,0 +1,21 @@
+adapters:
+  limelightDigital:
+    endpoint: http://{{Host}}.ortb.net/openrtb/{{PublisherID}}
```

Comment:
@bretg Is this dynamic endpoint host ok for you?

Reply:
Yes, partially dynamic is ok - they're on the list https://github.com/prebid/prebid-server/issues/2612